2-mandatory/4-space-colonies.js

```diff
@@ -15,7 +15,11 @@
   
 */
 
-function getSettlers() {}
+function getSettlers(voyagers) {
+  return voyagers.filter(
+    (voyager) => voyager.includes("family") && voyager[0] === "A"
```

Comment:
This works, could also use `string.endsWith` instead of `includes` / `string.startsWith` instead of `voyager[0]`